playbooks/deprovision.yaml

```diff
@@ -12,81 +12,11 @@
   - fail: msg="Bailing out. this play requires 'THOTH_NAMESPACE'"
     when: THOTH_NAMESPACE is not defined
 
-  - name: "deleting Thoth Templates"
-    command: "oc delete template {{ item }} --namespace {{ THOTH_NAMESPACE }}"
-    with_items:
-      - "thoth-core"
-      - "thoth-core-secret"
-      - "thoth-core-configmap"
-      - "janusgraph"
-      - "result-api-deployment"
-      - "result-api-buildconfig"
-      - "cleanup-buildconfig"
-      - "cleanup-cronjob"
-      - "graph-refresh-buildconfig"
-      - "graph-refresh-cronjob"
-      - "graph-sync-buildconfig"
-      - "graph-sync-cronjob"
-      - "package-extract-buildconfig"
-      - "package-releases-buildconfig"
-      - "package-releases-cronjob"
-      - "cleanup-imagestream"
-      - "graph-refresh-imagestream"
-      - "graph-sync-imagestream"
-      - "package-releases-imagestream"
-    ignore_errors: true
+  - name: Deprovision Thoth
+    command: "oc delete bc,configmap,cronjob,dc,is,pod,role,rolebinding,route,\
```

Comment:
if you deleted the dc all the pods should be gone...

Reply:
> if you deleted the dc all the pods should be gone...

But only for objects that have dc defined. When we run solver, adviser, package extract, these are run dynamically from user-api and do not have dc defined => these pods will not be deleted.